### ISSUE TITLE
Inconsistent electronic states for atomic carbon

### ISSUE THREAD

**Issue by rwest** [ReactionMechanismGenerator/RMG-database#171]
Running `grep '1 *C *u[0-9] *p[0-9] *c[0-9] *$' -B4 -A1 -r .` in the database shows a range of choices that have been made for the multiplicity and electronic structure of atomic carbon. In a couple of cases this has obviously been thought about , for example `primaryThermoLibrary.py` contains both
```
    label = "C(S)",
    molecule = 
"""
1 C u0 p2 c0
""",
```
and 
```
    label = "C(T)",
    molecule = 
"""
multiplicity 3
1 C u2 p1 c0
""",
```
(the chosen labels make it clear that one was meant to be singlet and one triplet)
however, some of them appear to be auto-generated adjacency lists in the quintuplet state,
which is probably not right.

We should check each of these:

```
./kinetics/libraries/Dooley/C1/dictionary.txt-C
./kinetics/libraries/Dooley/C1/dictionary.txt-multiplicity 5
./kinetics/libraries/Dooley/C1/dictionary.txt:1 C u4 p0 c0
./kinetics/libraries/Dooley/C1/dictionary.txt-
--
--
./kinetics/libraries/Dooley/methylformate/dictionary.txt-C
./kinetics/libraries/Dooley/methylformate/dictionary.txt-multiplicity 5
./kinetics/libraries/Dooley/methylformate/dictionary.txt:1 C u4 p0 c0
./kinetics/libraries/Dooley/methylformate/dictionary.txt-
--
--
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt-C
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt-multiplicity 5
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt:1 C u4 p0 c0
./kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt-
--
--
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt-C
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt-multiplicity 5
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt:1 C u4 p0 c0
./kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt-
--
--
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt-C(T)
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt-multiplicity 3
./kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt:1 C u2 p1 c0
--
--
./kinetics/libraries/FFCM1(-)/dictionary.txt-C
./kinetics/libraries/FFCM1(-)/dictionary.txt-multiplicity 5
./kinetics/libraries/FFCM1(-)/dictionary.txt:1 C u4 p0 c0
--
--
./kinetics/libraries/GRI-Mech3.0/dictionary.txt-C
./kinetics/libraries/GRI-Mech3.0/dictionary.txt-multiplicity 3
./kinetics/libraries/GRI-Mech3.0/dictionary.txt:1 C u2 p1 c0
--
--
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt-C(T)
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt-multiplicity 3
./kinetics/libraries/GRI-Mech3.0-N/dictionary.txt:1 C u2 p1 c0
--
--
./kinetics/libraries/JetSurF0.2/dictionary.txt-C
./kinetics/libraries/JetSurF0.2/dictionary.txt-multiplicity 5
./kinetics/libraries/JetSurF0.2/dictionary.txt:1 C u4 p0 c0
--
--
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt-C(T)
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt-multiplicity 3
./kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt:1 C u2 p1 c0
--
--
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt-C(T)
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt-multiplicity 3
./kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt:1 C u2 p1 c0
--
--
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt-C(T)
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt-multiplicity 3
./kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt:1 C u2 p1 c0
--
--
./thermo/libraries/Chernov.py-    label = "C",
./thermo/libraries/Chernov.py-    molecule = 
./thermo/libraries/Chernov.py-"""
./thermo/libraries/Chernov.py:1 C u0 p2 c0
./thermo/libraries/Chernov.py-""",
--
--
./thermo/libraries/FFCM1(-).py-    label = "C",
./thermo/libraries/FFCM1(-).py-    molecule = 
./thermo/libraries/FFCM1(-).py-"""
./thermo/libraries/FFCM1(-).py-multiplicity 5
./thermo/libraries/FFCM1(-).py:1 C u4 p0 c0
./thermo/libraries/FFCM1(-).py-""",
--
--
./thermo/libraries/GRI-Mech3.0-N.py-    label = "C(T)",
./thermo/libraries/GRI-Mech3.0-N.py-    molecule = 
./thermo/libraries/GRI-Mech3.0-N.py-"""
./thermo/libraries/GRI-Mech3.0-N.py-multiplicity 3
./thermo/libraries/GRI-Mech3.0-N.py:1 C u2 p1 c0
./thermo/libraries/GRI-Mech3.0-N.py-""",
--
--
./thermo/libraries/GRI-Mech3.0.py-    label = "C(T)",
./thermo/libraries/GRI-Mech3.0.py-    molecule = 
./thermo/libraries/GRI-Mech3.0.py-"""
./thermo/libraries/GRI-Mech3.0.py-multiplicity 3
./thermo/libraries/GRI-Mech3.0.py:1 C u2 p1 c0
./thermo/libraries/GRI-Mech3.0.py-""",
--
--
./thermo/libraries/JetSurF0.2.py-    label = "C",
./thermo/libraries/JetSurF0.2.py-    molecule = 
./thermo/libraries/JetSurF0.2.py-"""
./thermo/libraries/JetSurF0.2.py-multiplicity 5
./thermo/libraries/JetSurF0.2.py:1 C u4 p0 c0
./thermo/libraries/JetSurF0.2.py-""",
--
--
./thermo/libraries/Narayanaswamy.py-    label = "C(13)",
./thermo/libraries/Narayanaswamy.py-    molecule = 
./thermo/libraries/Narayanaswamy.py-"""
./thermo/libraries/Narayanaswamy.py:1 C u0 p2 c0
./thermo/libraries/Narayanaswamy.py-""",
--
--
./thermo/libraries/NISTThermoLibrary.py-    label = "C(T)",
./thermo/libraries/NISTThermoLibrary.py-    molecule = 
./thermo/libraries/NISTThermoLibrary.py-"""
./thermo/libraries/NISTThermoLibrary.py-multiplicity 3
./thermo/libraries/NISTThermoLibrary.py:1 C u2 p1 c0
./thermo/libraries/NISTThermoLibrary.py-""",
--
--
./thermo/libraries/primaryThermoLibrary.py-    label = "C(S)",
./thermo/libraries/primaryThermoLibrary.py-    molecule = 
./thermo/libraries/primaryThermoLibrary.py-"""
./thermo/libraries/primaryThermoLibrary.py:1 C u0 p2 c0
./thermo/libraries/primaryThermoLibrary.py-""",
--
--
./thermo/libraries/primaryThermoLibrary.py-    label = "C(T)",
./thermo/libraries/primaryThermoLibrary.py-    molecule = 
./thermo/libraries/primaryThermoLibrary.py-"""
./thermo/libraries/primaryThermoLibrary.py-multiplicity 3
./thermo/libraries/primaryThermoLibrary.py:1 C u2 p1 c0
./thermo/libraries/primaryThermoLibrary.py-""",
--
--
./thermo/libraries/USC-Mech-ii.py-    label = "C(T)",
./thermo/libraries/USC-Mech-ii.py-    molecule = 
./thermo/libraries/USC-Mech-ii.py-"""
./thermo/libraries/USC-Mech-ii.py-multiplicity 3
./thermo/libraries/USC-Mech-ii.py:1 C u2 p1 c0
./thermo/libraries/USC-Mech-ii.py-""",
--
--
./transport/libraries/GRI-Mech.py-    label = "C(T)",
./transport/libraries/GRI-Mech.py-    molecule = 
./transport/libraries/GRI-Mech.py-"""
./transport/libraries/GRI-Mech.py-multiplicity 3
./transport/libraries/GRI-Mech.py:1 C u2 p1 c0
./transport/libraries/GRI-Mech.py-""",
--
--
./transport/libraries/PrimaryTransportLibrary.py-    label = "C(S)",
./transport/libraries/PrimaryTransportLibrary.py-    molecule = 
./transport/libraries/PrimaryTransportLibrary.py-"""
./transport/libraries/PrimaryTransportLibrary.py:1 C u0 p2 c0
./transport/libraries/PrimaryTransportLibrary.py-""",
```

(Also, the SMILES interpretation and generation is broken for these, but that's an RMG-Py issue that I'll open later)

Check these off when checked (or corrected):

 - [x] kinetics/libraries/Dooley/C1/dictionary.txt:1 C u4 p0 c0
 - [x] kinetics/libraries/Dooley/methylformate/dictionary.txt:1 C u4 p0 c0
 - [x] kinetics/libraries/Dooley/methylformate_all_ARHEbathgas/dictionary.txt:1 C u4 p0 c0
 - [x] kinetics/libraries/Dooley/methylformate_all_N2bathgas/dictionary.txt:1 C u4 p0 c0
 - [x] kinetics/libraries/ERC-FoundationFuelv0.9/dictionary.txt:1 C u2 p1 c0
 - [x] kinetics/libraries/FFCM1(-)/dictionary.txt:1 C u4 p0 c0
 - [x] kinetics/libraries/GRI-Mech3.0/dictionary.txt:1 C u2 p1 c0
 - [x] kinetics/libraries/GRI-Mech3.0-N/dictionary.txt:1 C u2 p1 c0
 - [x] kinetics/libraries/JetSurF0.2/dictionary.txt:1 C u4 p0 c0
 - [x] kinetics/libraries/Nitrogen_Dean_and_Bozzelli/dictionary.txt:1 C u2 p1 c0
 - [x] kinetics/libraries/Nitrogen_Glarborg_Gimenez_et_al/dictionary.txt:1 C u2 p1 c0
 - [x] kinetics/libraries/Nitrogen_Glarborg_Zhang_et_al/dictionary.txt:1 C u2 p1 c0
 - [x] thermo/libraries/Chernov.py:1 C u0 p2 c0
 - [x] thermo/libraries/FFCM1(-).py:1 C u4 p0 c0
 - [x] thermo/libraries/GRI-Mech3.0-N.py:1 C u2 p1 c0
 - [x] thermo/libraries/GRI-Mech3.0.py:1 C u2 p1 c0
 - [x] thermo/libraries/JetSurF0.2.py:1 C u4 p0 c0
 - [x] thermo/libraries/Narayanaswamy.py:1 C u0 p2 c0
 - [x] thermo/libraries/NISTThermoLibrary.py:1 C u2 p1 c0
 - [x] thermo/libraries/primaryThermoLibrary.py:1 C u0 p2 c0
 - [x] thermo/libraries/primaryThermoLibrary.py:1 C u2 p1 c0
 - [x] thermo/libraries/USC-Mech-ii.py:1 C u2 p1 c0
 - [x] transport/libraries/GRI-Mech.py:1 C u2 p1 c0
 - [x] transport/libraries/PrimaryTransportLibrary.py:1 C u0 p2 c0

**Comment by alongd** [comment]
I think this can be closed after https://github.com/ReactionMechanismGenerator/RMG-database/pull/180 is merged.

**Comment by rwest** [comment]
Thanks Alon!

**Comment by KEHANG** [comment]
fixed by #180
